libmozevent/bus.py

```diff
@@ -111,19 +111,20 @@ def add_queue(self, name, mp=False, redis=False, maxsize=-1):
 
             return await _get()
 
-    async def run(self, method, input_name, output_name=None, sequential=True):
+    async def run(self, method, input_name, output_names=[], sequential=True):
```

Comment:
Nit: Use typing to check output_names is a list

Reply:
I've added type hints for all parameters in 01c4a51.